public/documents/fari-games/charge-srd.md

```diff
@@ -521,17 +598,18 @@ Once the mission is over, the PCs go back to their H.Q., and the loop starts ane
 > - _Finding dirt on the government to expose corruption._
 > - _Convincing the real king to take back what is theirs._
 > - _Finding money to hire mercenaries to fight an eventual war._
+>
+> Designing world forces is an efficient and simple way to fill the world with interesting factions that the players will be able to interact with.
 
-
-# Attachment | Extras
+# Attachment Extra | Extras
 
 If you want to resist a `consequence`, but don’t want to make a `determination roll`, you can decide to break one of your `attachment`. Doing so cancels the consequence outright.
```

Comment:
Should there be a bit more here to talk about how you might have attachments before jumping into how to use them? Maybe pull in first two paragraphs of the fully written extra like:

```suggestion
Characters are defined using `details` to represent who they are, what they are like and who they are linked to. By default, `details` don't have any mechanical uses in the game apart from providing narrative liberties.

An `attachment` is a character `detail` that you can choose to temporarily lose to resist an incoming `consequence`. If you want to resist a `consequence`, but don’t want to make a `determination roll`, you can decide to break one of your `attachment`. Doing so cancels the consequence outright.
```

Reply:
That's a very good suggestion. I love it